src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -111,6 +168,65 @@
                     "description": "Milliseconds since last update from peer"
                 }
             }
+        },
+        "self_test_report": {
+            "id": "self_test_report",
+            "description": "Self test single test report",
+            "properties": {
+                "disk_test_stoptime": {
+                    "type": "long",
+                    "description": "time to finish stub disk test"
+                },
+                "network_test_stoptime": {
+                    "type": "long",
+                    "description": "time to finish stub network test"
+                },
+                "failures": {
+                    "type": "boolean",
+                    "description": "True if any failures occurred"
+                }
+            }
+        },
+        "self_test_node_status": {
+            "id": "self_test_node_status",
+            "description": "Self test status",
+            "properties": {
+                "node_id": {
+                    "type": "long",
+                    "description": "node id"
+                },
+                "status": {
+                    "type": "long",
+                    "description": "Running or idle status"
+                },
+                "was_available": {
+                    "type": "boolean",
+                    "description": "Did the node have a failure"
+                },
+                "results": {
+                    "type": "self_test_report",
+                    "description": "test results from node"
+                }
+            }
+        },
+        "self_test_response": {
+            "id": "self_test_response",
+            "description": "Self test response",
```

Comment:
This description is a bit nondescript :-)

It's a response, but from what?  Should it be self_test_status_response or something?

Reply:
type removed